bin/truffleruby

```diff
@@ -24,6 +24,30 @@ function mtime {
     fi
 }
 
+# Prefix any option in $@ with -J:, values are just copied over
+function prefix_options {
+    local result=()
+    for opt in "$@"
+    do
+        case $opt in
+            -*)
+                result+=("-J:${opt} ") ;;
```

Comment:
No it does not, I am looking at the spaces issue right now. Right the space should go away.